assigner/baserepo.py

```diff
@@ -422,7 +456,7 @@ def new(cls, base_repo, semester, section, username, token):
         return cls.from_url(result["http_url_to_repo"], token)
 
     @classmethod
-    def name(cls, semester, section, assignment, user):
+    def build_name(cls, semester, section, assignment, user):
```

Comment:
This method had some weird conflict with Repo.name set in __init__. I highly doubt this ever caused a bug.

Reply:
It wouldn't work anyway... functions were missing args.